adaptive/learner/average_learner.py

```diff
@@ -44,6 +46,8 @@ def __init__(self, function, atol=None, rtol=None):
         self.atol = atol
         self.rtol = rtol
         self.npoints = 0
+        # Cannot estimate standard deviation with less than 2 points.
```

Comment:
```suggestion
        # Cannot estimate standard deviation with fewer than 2 points.
```

Sorry, I'll be pedantic here.